dask/dataframe/groupby.py

```diff
@@ -1629,6 +1703,34 @@ def mean(self, split_every=None, split_out=1, shuffle=None):
             c = c[s.columns]
         return s / c
 
+    @derived_from(pd.core.groupby.GroupBy)
+    def median(self, split_every=None, split_out=1):
```

Comment:
This doesn't have a `shuffle=` parameter like other groupby methods do. Is `shuffle="disk"` or `"p2p"` not valid here?

Reply:
Good point! I had removed `shuffle` as an option since this function *must* shuffle (`False` is not valid), but you are completely correct that the use can still choose a shuffling algorithm. I'll make that change here.